pessimistic-transaction.md

```diff
@@ -1,15 +1,21 @@
 ---
+<<<<<<< HEAD
```

Comment:
@qiancai Please resolve the conflict and merge this PR. Thanks!

Reply:
Thanks for the reminder. The conflict has been resolved.